spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +52,24 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+def test_copy_paste_files_or_paths(qtbot, tmpdir):
+    tmpdir.join('script.py').ensure()
+    window = DirView()
+    window.show()
+    qtbot.addWidget(window)
```

Comment:
Can you make the lines from the beginning of the test function to here a fixture, like the others above?

Reply:
LGTM